topmodel/README.md

```diff
@@ -0,0 +1,21 @@
+# Topographic Index Statistics for TOPMODEL Runoff
+This artifact includes topographic index statistics derived from a high-resolution map of topographic index, found here:
+Marthews, T.R., Dadson, S.J., Lehner, B., Abele, S., Gedney, N. (2015). High-resolution global topographic index values. NERC Environmental Information Data Centre. (Dataset). https://doi.org/10.5285/6b0c4358-2bf3-4924-aa8f-793d468b92bev
+
+This map was produced as described in the following:
+ Marthews, T. R., Dadson, S. J., Lehner, B., Abele, S., and Gedney, N.: High-resolution global topographic index values for use in large-scale hydrological modelling, Hydrol. Earth Syst. Sci., 19, 91–104, https://doi.org/10.5194/hess-19-91-2015, 2015.
+
+The raw data can be downloaded after making an account with the UK Centre for Ecology and Hydrology.
```

Comment:
Can a link be included here for what exactly is the raw data is being downloaded? Would it be possible to automate it? Also, can we include instructions for how to make the artifact?

Reply:
I added the direct download link, but it requires an account, so we cannot download it in an automated fashion. I also left instructions for how to make the artifact.